client/components/site-thumbnail.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+export default React.createClass({
+  propTypes: {
+    url: React.PropTypes.string
+  },
+  render () {
+    let block = {
+      position: 'absolute',
+      display: 'block',
+      height: '380',
+      width: '100%',
+      background: 'transparent'
+    }
```

Comment:
I think a CSS `pointer-events: none;` on the iframe element will achieve the same effect much more succinctly.